### PR TITLE
Grammar corrections and build path fix.

### PR DESCRIPTION
**Pull request by xero-lib**

### CODE REVIEW

File: lang/en-gb.xml [hunk 1]
```diff
@@ -275,8 +275,8 @@ Connect a controller that is compatible with SDL2.</not_connected>
 			<save name="Save">
 				<cancel_saving>Do you want to cancel saving?</cancel_saving>
 				<could_not_save>Could not save the file.
-There is not enough free space on the memory card.</could_not_save>
-				<not_free_space>There is not enough free space on the memory card.</not_free_space>
+There is not enough free space on the disk.</could_not_save>
+				<not_free_space>There is not enough free space on the disk.</not_free_space>
```
Comment (Zangetsu38) [inline lang/en-gb.xml:279]: revert

File: lang/en-gb.xml [hunk 2]
```diff
@@ -439,11 +438,11 @@ It is also generally recommended for GUI.</firmware_font_package_note>
 	<trophy_collection>
 		<search>Search</search>
 		<delete_trophy>Delete Trophy</delete_trophy>
-		<trophy_deleted>This trophy information saved on this user will be deleted.</trophy_deleted>
+		<trophy_deleted>Trophy information saved for this user will be deleted.</trophy_deleted>
```
Comment (Zangetsu38) [inline lang/en-gb.xml:441]: revert, official string is
"This trophy information saved on this system will be deleted."

File: lang/en-gb.xml [hunk 3]
```diff
@@ -439,11 +438,11 @@ It is also generally recommended for GUI.</firmware_font_package_note>
...
 		<details>Details</details>
 		<earned>Earned</earned>
 		<name>Name</name>
-		<no_trophies>There are no trophies.
```
Comment (Zangetsu38) [inline lang/en-gb.xml:446]: add user if you want, but no touch string, is official

File: lang/en-gb.xml [hunk 4]
```diff
@@ -459,11 +458,11 @@ You can earn trophies by using an application that supports the trophy feature.<
 		<create_user>Create User</create_user>
 		<user_created>The following user has been created</user_created>
 		<edit_user>Edit User</edit_user>
-		<user_name_used>! This name is already in use.</user_name_used>
+		<user_name_used>This name is already in use.</user_name_used>
 		<delete_user>Delete User</delete_user>
 		<user_delete>Select the user you want to delete.</user_delete>
-		<user_delete_msg>The following user will be deleted.</user_delete_msg>
-		<user_delete_message>If you delete the user, that user's saved data, trophies will be deleted.</user_delete_message>
+		<user_delete_msg>The following user will be deleted:</user_delete_msg>
```
Comment (Zangetsu38) [inline lang/en-gb.xml:464]: revert

File: lang/en-gb.xml [hunk 5]
```diff
@@ -459,11 +458,11 @@ You can earn trophies by using an application that supports the trophy feature.<
...
+		<user_delete_message>If you delete the user, that user's saved data (trophies, etc) will be deleted.</user_delete_message>
```
Comment (Zangetsu38) [inline lang/en-gb.xml:465]: same

File: lang/en-gb.xml [hunk 6]
```diff
@@ -483,7 +482,7 @@ After the download is complete, Vita3K will restart automatically and then insta
 		<not_complete_update>Could not complete the update.</not_complete_update>
 		<next>Next</next>
 		<update_vita3k>Do you want to update Vita3K?</update_vita3k>
-		<later_version_already_installed>The later version of the Vita3K software is already installed.</later_version_already_installed>
+		<later_version_already_installed>This version of the Vita3K software is already installed.</later_version_already_installed>
```
Comment (Zangetsu38) [inline lang/en-gb.xml:485]: revert is official string

**Review by Zangetsu38 — CHANGES_REQUESTED** [review]
done all like i said, same for en-gb and no forget, if you touch on english lang, you need also change st'ring corresponding inside lang/state.h

**Comment by Croden1999** [comment]
u can changed install dialog, not changed other strings, there are based on PSVita/PS4 language.

File: lang/en-gb.xml [hunk 7]
```diff
@@ -289,7 +289,7 @@ Do not power off the system or close the application.</warning_saving>
 	</dialog>
 
 	<game_data>
-		<app_close>The following application will close.</app_close>
+		<app_close>The following application will close:</app_close>
```
Comment (Zangetsu38) [inline lang/en-gb.xml:292]: Revert this i have said

File: vita3k/lang/include/lang/state.h [hunk 8]
```diff
@@ -150,7 +150,7 @@ struct LangState {
         { "history_version", "Version {}" },
         { "information", "Information" },
         { "app_delete", "This application and all related data, including saved data, will be deleted." },
-        { "save_delete", "Do you want to delete this saved data ?" },
+        { "save_delete", "Do you want to delete this saved data?" },
```
Comment (Zangetsu38) [inline vita3k/lang/include/lang/state.h:153]: Good catch, copy past have added space

**Comment by xero-lib** [comment]
Missed that revert, fixed now.

File: lang/en.xml [hunk 9]
```diff
@@ -289,7 +289,7 @@ Do not power off the system or close the application.</warning_saving>
 	</dialog>
 
 	<game_data>
-		<app_close>The following application will close.</app_close>
+		<app_close>The following application will close:</app_close>
```
Comment (Zangetsu38) [inline lang/en.xml:292]: forget revert it

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**